mvlearn/embed/mvmds.py

```diff
@@ -41,6 +41,9 @@ class MVMDS(BaseEmbed):
 
     num_iter: int (positive), default=15
         Number of iterations stepwise estimation goes through.
+        
+    distance: bool, default=False
```

Comment:
look at how sklearn handles this - they usually have an attribute called `dissimilarity` or `metric`, and you can input things like "euclidean" or "precomputed". Where "precomputed" is what you describe here.

Reply:
i would try to follow their API as much as possible

Reply:
yeah definitely try to match sklearn for this kind of thing